lwcomponent/catalog.go

```diff
@@ -200,6 +200,21 @@ func (c *Catalog) Install(component *CDKComponent) (err error) {
 	return component.stage.Commit(componentDir)
 }
 
+// Delete a CDKComponent
+//
+// Remove the Component install directory and all sub-directory.  This function will not return an
+// error if the Component is not installed.
+func (c *Catalog) Delete(component *CDKComponent) (err error) {
+	componentDir, err := componentDirectory(component.Name)
+	if err != nil {
+		return
+	}
+
+	os.RemoveAll(componentDir)
+
+	return
```

Comment:
Should the result of component dir not found be logged even if it's not being return as an error?

Reply:
Does make sense and is consistent with old functionality. Added.